pom.xml

```diff
@@ -103,7 +103,7 @@
     <timestamp>${maven.build.timestamp}</timestamp>
     <maven.build.timestamp.format>yyyy-MM-dd HH:mm</maven.build.timestamp.format>
 
-    <beam.version>0.6.0</beam.version>
+    <beam.version>2.0.0</beam.version>
 
     <dataflow.release_name>Google Cloud Dataflow SDK for Java</dataflow.release_name>
     <dataflow.container_version>${project.version}-20170317</dataflow.container_version>
```

Comment:
We decided that option 2 was what we were going to do. I will add this to the documentation.